pandas/io/formats/style.py

```diff
@@ -764,22 +833,40 @@ def set_caption(self, caption):
 
     def set_table_styles(self, table_styles):
         """
-        Set the table styles on a Styler. These are placed in a
-        ``<style>`` tag before the generated HTML table.
+        Set table styles. These are placed in a `<style>` tag before the
+        generated HTML table.
 
         Parameters
         ----------
         table_styles: list
-            Each individual table_style should be a dictionary with
-            ``selector`` and ``props`` keys. ``selector`` should be a CSS
+            Each list item should be a dictionary with
+            `selector` and `props` keys. `selector` should be a CSS
             selector that the style will be applied to (automatically
-            prefixed by the table's UUID) and ``props`` should be a list of
-            tuples with ``(attribute, value)``.
+            prefixed by the table's UUID) and `props` should be a list of
+            tuples with `(attribute, value)`.
 
         Returns
         -------
         self : Styler
 
+        Notes
+        -----
+        Entries are rendered with CSS selectors according to gross properties
+        of their position and content:
+        * Index and Column names include ``index_name`` and ``level<k>``
```

Comment:
`set_table_styles` doesn't seem like the appropriate place for this section. Perhaps `.render`?